packages/itwinui-css/src/global.scss

```diff
@@ -61,3 +61,79 @@
 }
 
 @include mixins.animations;
+
+:where(.iui-field) {
```

Comment:
Not convinced this should be within `global.scss`.

Reply:
I'm not sure where else it could go, since it is not related to a specific component. Maybe it would make sense to put it in `input.scss`?

Reply:
> Not convinced this should be within `global.scss`.

If not in `global.scss`, maybe we can create a new file called `field.scss` in `src/utils`?

Reply:
This should be in its own file, which can be imported in `all.scss`.

Reply:
Moved out of `global.scss` to `/src/utils/field.scss` and imported in `all.scss`.

Reply:
The `utils/` folder already creates a `utils.css` file (using [`utils.scss`](https://github.com/iTwin/iTwinUI/blob/main/packages/itwinui-css/src/utils/utils.scss)), so this one should live outside of `utils/` because we want it to be near the top of `all.scss`.

Reply:
Yeah `src/field.scss` is fine for now. We can think about organizing folders in the future when we introduce internal cascade layers.

Reply:
Moved to `src/field.scss` and updated `all.scss` to reflect that.